src/components/YourModule.js

```diff
@@ -0,0 +1,343 @@
+      assets: [],
+      toAddress: (props.scannerState ? props.scannerState.toAddress : ""),
+    }
+  }
+
+  componentDidMount(){
+    console.log("YOUR MODULE MOUNTED, PROPS:",this.props)
+    /*
+        -- LOAD YOUR CONTRACT --
+        Contract files loaded from:
+        src/contracts/YourContract.abi
+        src/contracts/YourContract.address
+        src/contracts/YourContract.blocknumber.js // the block number it was deployed at (for efficient event loading)
+        src/contracts/YourContract.bytecode.js // if you want to deploy the contract from the module (see deployYourContract())
+    */
+    this.setState({
+     YourContract: this.props.contractLoader("YourContract")
+    },()=>{
+     console.log("YOURCONTRACT IS LOADED:",this.state.YourContract)
+    })
+
+    setInterval(this.pollInterval.bind(this),2500)
+    setTimeout(this.pollInterval.bind(this),30)
+    
+    axios
+    .get(
+      `https://api.opensea.io/api/v1/assets/?owner=${
+        this.props.address
+      }&order_by=current_price&order_direction=asc`
+    )
+    .then(response => {
+      console.log("Data#### ", response.data.assets);
+      this.setState({
+        assets: response.data.assets
+      });
+    });
+  }
+
+  async pollInterval(){
+    console.log("POLL")
+    if(this.state && this.state.YourContract){
+      let yourVar = await this.state.YourContract.YourVar().call()
+      let yourContractBalance = await this.props.web3.eth.getBalance(this.state.YourContract._address)
+      //let ensName = await this.props.ensLookup("austingriffith.eth")
+      let mainnetBlockNumber = await this.props.mainnetweb3.eth.getBlockNumber()
+      let xdaiBlockNumber = await this.props.xdaiweb3.eth.getBlockNumber()
+      yourContractBalance = this.props.web3.utils.fromWei(yourContractBalance,'ether')
+      this.setState({yourVar,yourContractBalance,mainnetBlockNumber,xdaiBlockNumber})
+
+    }
+  }
+
+  clicked(name){
+    console.log("secondary button "+name+" was clicked")
+    /*
+    Time to make a transaction with YourContract!
+    */
+    this.props.tx(this.state.YourContract.updateVar(name),120000,0,0,(result)=>{
+      console.log(result)
+    })
+
+  }
+  deployYourContract() {
+    console.log("Deploying YourContract...")
+    //
+    //  as noted above you need src/contracts/YourContract.bytecode.js
+    //  to be there for this to work:
+    //
+    let code = require("../contracts/YourContract.bytecode.js")
+    this.props.tx(this.state.YourContract._contract.deploy({data:code}),640000,(receipt)=>{
+      let yourContract = this.props.contractLoader("YourContract",receipt.contractAddress)
+      this.setState({ YourContract: yourContract})
+    })
+  }
+  render(){
+
+    if(!this.state.YourContract){
+      return (
+        <div>
+          LOADING YOURCONTRACT...
+        </div>
+      )
+    }
+
+    return (
+      <div>
+        <div className="form-group w-100">
+
+          <div style={{width:"100%",textAlign:"center"}}>
```

Comment:
Is all of this information below really useful to a user? I think we can remove it.